### ISSUE TITLE
Block pipe arguments improperly highlighted (or not at all.)

### ISSUE THREAD

**Issue by envygeeks** [atom/language-ruby#93]
```ruby
helper :get_hash do |key = (task_options[:get] || task_options[:set]), hash = config, create: false|
  # Do Work
end
```

To be fair this seems to also be a problem with Pygments too.

**Comment by mamenama** [comment]
Added code in the grammar in 17c43c0029c60edc3eced9675451293d819caba3 that will prevent this issue.

**Comment by lee-dohm** [comment]
It appears this is also part of #113

**Comment by 50Wliu** [comment]
Fixed by #113.
